loopchain/consensus/runner.py

```diff
@@ -273,6 +273,10 @@ async def _round_start(self, event: RoundEndEvent):
         mediator = self.event_system.get_mediator(DelayedEventMediator)
         mediator.execute(0.5, round_start_event)
 
+    def update_status(self, peer: str, height: int):
+        # TODO: Update height info
```

Comment:
Is it intention of you that method doesn't implement?

Reply:
You right, It is intended abstraction, which is entirely up to @noveljava in another PR.